GitUI/UserControls/RevisionGrid/RevisionDataGridView.cs

```diff
@@ -567,16 +572,18 @@ private void SelectRowsIfReady(int rowCount)
                 return;
             }
 
+            CalculateGraphIndices();
```

Comment:
Always calculating the graph indices has a performance impact: This is done every time SetRowCount() is called. (The global variable is not really needed with this setup, just FoundAnyToBeSelected to be changed or renamed).
(Not executing at every row but every "chunk of grid rows loaded").
Revisions are loaded before the grid rows.

So I want to revert to the previous handling.

Reply:
> So I want to revert to the previous handling.

Sure thing

Reply:
I changed the handling to use Lazy<>, to not trigger similar concerns in the future.